lib/kitto/router.ex

```diff
@@ -21,6 +21,20 @@ defmodule Kitto.Router do
     end
   end
 
+  post "dashboards" do
+    {:ok, body, conn} = read_body conn
+    command = body |> Poison.decode! |> Map.put_new("dashboard", "*")
+    Kitto.Notifier.broadcast! "_kitto", command
+    conn |> send_resp(204, "")
+  end
+
+  post "dashboards/:id" do
+    {:ok, body, conn} = read_body conn
+    command = body |> Poison.decode! |> Map.put("dashboard", id)
```

Comment:
At some point we'll have to swap all our `Poison.decode!` with their non-bang version.
When one posts invalid JSON the error output is quite illigible, like:

```
00:03:27.111 [error] Ranch listener Kitto.Router.HTTP had connection process started with :cowboy_protocol:start_link/4 at #PID<0.368.0> 
exit with reason: {{%Poison.SyntaxError{message: "Unexpected token: }", token: "}"}, 
[{Poison.Parser, :parse!, 2, [file: 'lib/poison/parser.ex', line: 56]}, {Poison, :decode!, 2, [file: 'lib/poison.ex', line: 83]}, {Kitto.Router, :"-do_match/4-fun-3-", 1, [file: 'lib/kitto/router.ex', line: 27]}, {Kitto.Router, :plug_builder_call, 2, 
[file: 'lib/kitto/router.ex', line: 1]}, 
{Plug.Adapters.Cowboy.Handler, :upgrade, 4, [file: 'lib/plug/adapters/cowboy/handler.ex', line: 15]}, 
{:cowboy_protocol, :execute, 4, [file: 'src/cowboy_protocol.erl', line: 442]}]}, {Kitto.Router, :call, [%Plug.Conn{adapter: {Plug.Adapters.Cowboy.Conn, :...}, assigns: %{}, before_send: [], body_params: %Plug.Conn.Unfetched{aspect: :body_params}, cookies: %Plug.Conn.Unfetched{aspect: :cookies}, halted: false, host: "localhost", method: "POST", owner: #PID<0.368.0>, params: %Plug.Conn.Unfetched{aspect: :params}, path_info: ["dashboards"], peer: {{127, 0, 0, 1}, 59194}, port: 4000, private: %{}, query_params: %Plug.Conn.Unfetched{aspect: :query_params}, query_string: "", remote_ip: {127, 0, 0, 1}, req_cookies: %Plug.Conn.Unfetched{aspect: :cookies}, req_headers: [{"host", "localhost:4000"}, {"user-agent", "curl/7.50.1"}, {"accept", "*/*"}, {"content-length", "41"}, {"content-type", "application/x-www-form-urlencoded"}], request_path: "/dashboards", resp_body: nil, resp_cookies: %{}, resp_headers: [{"cache-control", "max-age=0, private, must-revalidate"}], scheme: :http, script_name: [], secret_key_base: nil, state: :unset, status: nil}, []]}}
```